test/core-flags/infrastructure-logging.test.js

```diff
@@ -27,4 +28,15 @@ describe('infrastructure logging related flag', () => {
         expect(stderr).toContain("Compilation 'compiler' finished");
         expect(stdout).toContain(`level: 'log'`);
     });
+
+    it('should log used default config when level is log', () => {
+        const { exitCode, stderr, stdout } = run(__dirname, ['--infrastructure-logging-level', 'log']);
+
+        expect(exitCode).toBe(0);
+        const configPath = resolve(__dirname, './webpack.config.js');
+        expect(stderr).toContain("Compilation 'compiler' starting...");
+        expect(stderr).toContain(`Using config ${configPath}`);
+        expect(stderr).toContain("Compilation 'compiler' finished");
+        expect(stdout).toContain(`level: 'log'`);
+    });
```

Comment:
We don't need this test here, we already have test for `--infrastructure-logging-level`

Reply:
done remove